addons/event_booth_sale/models/event_booth_category.py

```diff
@@ -16,12 +16,15 @@ def _default_product_id(self):
 
     product_id = fields.Many2one(
         'product.product', string='Product', required=True,
-        domain=[('detailed_type', '=', 'event_booth')], default=_default_product_id)
-    price = fields.Float(string='Price', compute='_compute_price', digits='Product Price', readonly=False, store=True)
-    currency_id = fields.Many2one(related='product_id.currency_id')
+        domain=[('detailed_type', '=', 'event_booth')], default=_default_product_id,
+        groups="event.group_event_registration_desk")
+    price = fields.Float(
+        string='Price', compute='_compute_price', digits='Product Price', readonly=False,
+        store=True, groups="event.group_event_registration_desk")
+    currency_id = fields.Many2one(related='product_id.currency_id', groups="event.group_event_registration_desk")
     price_reduce = fields.Float(
         string='Price Reduce', compute='_compute_price_reduce',
-        compute_sudo=True, digits='Product Price')
+        compute_sudo=True, digits='Product Price', groups="event.group_event_registration_desk")
     image_1920 = fields.Image(compute='_compute_image_1920', readonly=False, store=True)
```

Comment:
Is there a "lighter" fix available for the stable version?

Reply:
Yes I could simply set a default image on the `src` attribute if it's an issue